circuit_maintenance_parser/parsers/windstream.py

```diff
@@ -0,0 +1,78 @@
+"""Windstream Fiber parser."""
+import logging
+from datetime import datetime, timezone
+import pytz
+
+from circuit_maintenance_parser.parser import Html, Impact, CircuitImpact, Status
+
+# pylint: disable=too-many-nested-blocks, too-many-branches
+
+logger = logging.getLogger(__name__)
+
+
+class HtmlParserWindstream1(Html):
+    """Notifications Parser for Windstream notifications."""
+
+    def parse_html(self, soup):
+        """Execute parsing."""
+        data = {}
+        data["circuits"] = []
+        impact = Impact("NO-IMPACT")
+        confirmation_words = [
+            "Demand Maintenance Notification",
+            "Planned Maintenance Notification",
+            "Emergency Maintenance Notification",
+        ]
+        cancellation_words = ["Postponed Maintenance Notification", "Cancelled Maintenance Notification"]
+
+        h1_tag = soup.find("h1")
+        if h1_tag.string.strip() == "Completed Maintenance Notification":
+            data["status"] = Status("COMPLETED")
+        elif any(keyword in h1_tag.string.strip() for keyword in confirmation_words):
+            data["status"] = Status("CONFIRMED")
+        elif h1_tag.string.strip() == "Updated Maintenance Notification":
+            data["status"] = Status("RE-SCHEDULED")
+        elif any(keyword in h1_tag.string.strip() for keyword in cancellation_words):
+            data["status"] = Status("CANCELLED")
+
+        div_tag = h1_tag.find_next_sibling("div")
+        summary_text = div_tag.get_text(separator="\n", strip=True)
+        summary_text = summary_text.split("\nDESCRIPTION OF MAINTENANCE")[0]
+
+        data["summary"] = summary_text
+
+        table = soup.find("table")
+        for row in table.find_all("tr"):
+            if len(row) < 2:
+                continue
+            cols = row.find_all("td")
+            header_tag = cols[0].string
+            if header_tag is None or header_tag == "Maintenance Address:":
+                continue
+            header_tag = header_tag.string.strip()
+            value_tag = cols[1].string.strip()
+            if header_tag == "WMT:":
+                data["maintenance_id"] = value_tag
+            elif "Date & Time:" in header_tag:
+                value_tag = value_tag.replace(" ET", "")
+                date_obj_est = datetime.strptime(value_tag, "%m/%d/%y %H:%M")
+                date_obj_est = pytz.timezone("US/Eastern").localize(date_obj_est)
```

Comment:
I guess that this may change depending on the customer's timezone. what do you think?

Reply:
Hmm I guess that is a good point.
I may only have maintenance email examples from Windstream for our circuits that belong in that region. I haven't got any other emails to base this information on.

Do you have any functions which use a timezone checker that I can replicate (based on the abbreviation used at the end of a datetime string)?
If not, I'll come up with my own :)

Reply:
no @aliex-13 , we don't have, but I think it would be a useful helper function

Reply:
Hey @chadell - I've pushed a new commit with a new helper function in the "utils.py" file. I'm not sure if this was the correct location but I could see similar functions there. Please let me know if I need to move it/fix it.

I've added all of the basic US related timezones to this function. 
We can easily add more as shown in the code.

Does this work for you? All of my tests pass.